web/src/features/fbaCalculator/RowManager.ts

```diff
@@ -121,6 +124,12 @@ export class RowManager {
       case SortByColumn.HFI: {
         return _.orderBy(tableRows, 'head_fire_intensity', order)
       }
+      case SortByColumn.CriticalHours4000: {
+        return _.orderBy(tableRows, 'critical_hours_hfi_4000.start', reverseOrder)
```

Comment:
Curious why we need to reverse the order for critical hours?

Reply:
good point - this is something that I should probably confirm with Tess. But basically, when sorting crit hours in descending order, I'd expect that the "worst" scenarios (i.e., the rows with the largest time range of critical hours) would appear first. If I don't reverse the order, then the blank critical hours get displayed first, because of how lodash compares null to a number

Reply:
is my logic correct @tessmgood - sorting critical hours in descending order means "put worst-case scenarios at the top of the list"?

Reply:
So my understanding of sort is just by the start value which is how it's sorted currently. This sort doesn't order by largest / smallest range since you could have a large range regardless of when the start time is.